### PR TITLE
Add a test case for export/import large file repo

### PR DESCRIPTION
**Pull request by vsedmik**
### Problem Statement
ISS export of a CV with more than 2^16 files was causing errors (see the issue below). It was fixed in product and needs coverage since it is customer facing.

### Solution
This PR adds a test case to test both, export and import, of such a large repo.

### Related Issues
https://issues.redhat.com/browse/SAT-25194

### Dependencies
https://github.com/SatelliteQE/fedorapeople-repos/pull/15
sat-qe jenkins MR#1440 (config related)

### PRT test Cases example
(both dependencies need to be merged first)
trigger: test-robottelo
pytest: tests/foreman/cli/test_satellitesync.py -k test_postive_export_import_large_cv

### CODE REVIEW

**Review by jyejare — APPROVED** [review]
Please make the change in CI for this change.

Edited: I see MR#1440 linked here. Thanks

**Review by ogajduse — APPROVED** [review]

**Comment by ogajduse** [comment]
Waiting on review by @SatelliteQE/team-phoenix.

File: conf/repos.yaml.template
Comment (vsedmik) [reply conf/repos.yaml.template:189]: Nope, it's just a "property" of that repo which is asserted by the very last assert.

**Review by vijaysawant — APPROVED** [review]
Ack

**Pull request merged**
